compiler/rustc_traits/src/normalize_erasing_regions.rs

```diff
@@ -20,6 +20,14 @@ crate fn provide(p: &mut Providers) {
         normalize_mir_const_after_erasing_regions: |tcx, goal| {
```

Comment:
Can this query be implemented in terms of `try_normalize_mir_const_after_erasing_regions`? What about `normalize_generic_arg_after_erasing_regions` and `try_normalize_generic_arg_after_erasing_regions`?

Reply:
Yes, both could. We would just have to unwrap the result and more explicitly document that these can panic if some type is not normalizable. Do you think this would improve caching?

Reply:
I think this is a good idea, but fine for followup. Add a FIXME please?